deltacat/compute/stats/basic.py

```diff
@@ -25,6 +25,23 @@ def collect(
         columns: Optional[List[str]] = None,
         stat_results_s3_bucket: Optional[str] = None,
         deltacat_storage=unimplemented_deltacat_storage) -> Dict[int, DeltaStats]:
+    """
+    Collects statistics on deltas, given a set of delta stream position ranges.
+
+    The source partition locator is a reference to the partition locator object tied to the given
+    delta stream positions. Columns can be optionally included to collect stats on specific columns,
```

Comment:
One important thing to note is that each tuple in the range set represents a bounded, closed interval. 

Also, even though we haven't done it historically, this would be a good place to start following the [Ray Code Style](https://docs.ray.io/en/latest/getting-involved.html#code-style) for docstrings. Eventually, I'd like to refactor all of our docstrings to follow the same style. 

Aside from creating a consistent look and appearance, this will also help with auto-generated API docs.

Reply:
I think this would be a good opportunity to apply Ray's docstring style 🙂 .